osu.Android/FodyWeavers.xsd

```diff
@@ -0,0 +1,34 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
+  <!-- This file was generated by Fody. Manual changes to this file will be lost when your project is rebuilt. -->
```

Comment:
Is there any reason to include the weavers schema files in the git tree ? From my past experience with Fody, these are usually `.gitignore`d (FWIW, they are included in the `.gitignore` file for Visual Studio projects) since they are regenerated on project rebuilding and don't serve any real purpose other than enumerating the config properties of the weavers.

So maybe this is an oversight ?

Reply:
This is already committed in the desktop project. See [`osu.Game` tree](https://github.com/ppy/osu/tree/master/osu.Game).

I'm fine with ignoring, but if ignoring is to be done, then it should be done consistently.

Reply:
Also see [the original discussion on this](https://github.com/ppy/osu/pull/11461#discussion_r619315597).

Reply:
oh, missed that one was already committed to the tree. nevermind then

Reply:
Looking at the build log the android ruleset test projects are complaining about not having those files, too. So we may want to ignore in the end to reduce noise. @peppy @smoogipoo thoughts?

Reply:
Deleting the existing ones and ignoring at a `.gitignore` level seems correct. Want to go ahead and do that in the PR?

Reply:
Sure, will do in a bit.

Reply:
I've removed and ignored all of the schema files.

As for the build warnings, I misunderstood them - they were about the `.xml` files and not the `.xsd` schemas, and it looks like the `.xml` ones are indeed intended to be committed to projects. Thankfully Fody [supports solution-level `FodyWeavers.xml`](https://github.com/Fody/Home/blob/master/pages/configuration.md#a-file-in-the-solution-directory), so I've moved the `osu.Game` file out to the solution directory to not duplicate it in every end-user project.